code/stable/gamephysics/SRS/GamePhysics_SRS.tex

```diff
@@ -856,13 +849,11 @@ \subsubsection{Data Definitions}
               \item{${\mathbf{I}_{\text{B}}}$ is the moment of inertia of rigid body B (kg$\text{m}^{2}$)}
               \end{symbDescription}
 \\ \midrule \\
-Notes & \hyperref[assumpOT]{A: objectTy}
-        
-        \hyperref[assumpOD]{A: objectDimension}
+Notes & All bodies are assumed to be rigid (from \hyperref[assumpOT]{A: objectTy}) and two-dimensional (from \hyperref[assumpOD]{A: objectDimension}).
```

Comment:
I would have to go through the details of the derivation to see if it generalizes to 3D, so for now, let us leave in the assumption that it is 2D.